region/maridia/outer.json

```diff
@@ -739,7 +739,30 @@
                 }
               ]
             }
-          ]
+          ],
+          "gModeImmobile": {
+            "requires": [
+              {"enemyDamage": {
+                "enemy": "Sciser",
+                "type": "contact",
+                "hits": 1
+              }},
+              {"or": [
+                "Bombs",
+                {"ammo": { "type": "PowerBomb", "count": 1 }},
+                {"enemyDamage": {
+                  "enemy": "Sciser",
+                  "type": "contact",
+                  "hits": 1
+                }}
+              ]}
+            ],
+            "note": "Start on the right side and carefully land on the right edge. While in artificial morph, carefully bomb over the crab to avoid a second hit.",
+            "devNote": [
+              "It may be possible to get hit on the left side, but it would be dependent on the strats coming in, sometimes you start too high to get hit.",
+              "All known gmode strats in this room require artificial morph, otherwise the crab could easily be jumped over."
```

Comment:
Springball.  I think your movement options are Grav+IBJ, or HJB+pseudo double springballjumps.
Bombing over the crab would use Gravity too but maybe you could kill it.

And I saw a clip of GCC getting the missile by standing inside the speed blocks.

Reply:
I dont believe we have a way to represent this immobile strat, so I am just going to delete it for now. This immobile strat only has extra requirements if the strats that use it need artificial morph, and only those that remain in the room. (Staying up, in morph, will require dealing with the crab, everything else won't)

In any case, there arent even any upward gmode strats yet.